sdks/python/apache_beam/runners/portability/portable_metrics.py

```diff
@@ -0,0 +1,74 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from __future__ import absolute_import
+
+from apache_beam.metrics import monitoring_infos
+from apache_beam.metrics.execution import MetricKey
+from apache_beam.metrics.metric import MetricName
+
+
+class ParseMonitoringInfoMixin(object):
+  @staticmethod
+  def from_monitoring_infos(monitoring_info_list, user_metrics_only=False):
+    """Groups MonitoringInfo objects into counters, distributions and gauges.
+
+    Args:
+      monitoring_info_list: An iterable of MonitoringInfo objects.
+      user_metrics_only: If true, includes user metrics only.
```

Comment:
Please add a Returns section to pydoc comment

Reply:
+1